README.md

```diff
@@ -5,6 +5,112 @@
+
+```diff
+
+return [
+    // ...
+-        Sulu\Bundle\SnippetBundle\SuluSnippetBundle::class => ['all' => true],
+-        Sulu\Bundle\ArticleBundle\SuluArticleBundle::class => ['all' => true],
+-        ONGR\ElasticsearchBundle\ONGRElasticsearchBundle::class => ['all' => true],
+-        Sulu\Bundle\SnippetBundle\SuluSnippetBundle::class => ['all' => true],
+
++        Sulu\Bundle\PhpcrMigrationBundle\SuluPhpcrMigrationBundle::class => ['all' => true],
++        Sulu\Content\Infrastructure\Symfony\HttpKernel\SuluContentBundle::class => ['all' => true],
++        Sulu\Messenger\Infrastructure\Symfony\HttpKernel\SuluMessengerBundle::class => ['all' => true],
++        Sulu\Article\Infrastructure\Symfony\HttpKernel\SuluArticleBundle::class => ['all' => true],
++        Sulu\Snippet\Infrastructure\Symfony\HttpKernel\SuluSnippetBundle::class => ['all' => true],
++        Sulu\Page\Infrastructure\Symfony\HttpKernel\SuluPageBundle::class => ['all' => true],
+```
+
+Configure the SuluPhpcrMigrationBundle in `config/packages/sulu_phpcr_migration.yaml`:
+
+> If you are currently using Jackrabbit, use the "jackrabbit://" based DSN string.
+> After the upgrade, Apache Jackrabbit is no longer used by Sulu’s new content storage and can be removed from
+> your projects in most situations.
+
+```yaml
+sulu_phpcr_migration:
+    # dbal://<dbalConnection>?workspace=<workspaceName>
+    # jackrabbit://<user>:<password>@<host>:<port>/server?workspace=<workspaceName>
+    #    DSN: "dbal://default?workspace=%env(PHPCR_WORKSPACE)%"
+    #    DSN: "jackrabbit://admin:admin@127.0.0.1:8080/server?workspace=%env(PHPCR_WORKSPACE)%"
+    DSN: "dbal://default?workspace=%env(PHPCR_WORKSPACE)%"
+    target:
+        dbal:
+            connection: default
+
+```
+
+The new content structure used in Sulu 3.0 requires that all the `resource_locators` or `route` properties must be 
+renamed to `url` in your templates.
+
+```diff
+-        <property name="routePath" type="route">
++        <property name="url" type="route">
+            <meta>
+                <title lang="en">Resourcelocator</title>
+                <title lang="de">Adresse</title>
+            </meta>
+
+            <tag name="sulu_article.article_route"/>
+        </property>
+```
+
+Additionally, the controller in the page/article templates has to be adjusted use the new controller 
+from the SuluContentBundle. Be aware that also your custom controllers have to be modified to extend from the new one.
+
+```diff
+-        <controller class="Sulu\Bundle\ArticleBundle\Controller\ArticleController"/>
+-        <controller>Sulu\Bundle\WebsiteBundle\Controller\DefaultController::indexAction</controller>
+-        <controller>Sulu\Content\UserInterface\Controller\Website\ContentController::indexAction</controller>
+```
+
+Now you have to update the database schema. You can use the following command to do that:
+
+```shell
+    php bin/adminconsole doctrine:schema:update --force
```

Comment:
Probably it would also make sense here, to insert all the mysql update statements, as we did that also in previous database changes